recipes/devices/families/x86.sh

```diff
@@ -160,17 +160,14 @@ device_chroot_tweaks_pre() {
   # Confirm anyway
   dpkg -i linux-image-*_"${ARCH}".deb
   rm linux-image-*_"${ARCH}".deb
-  log "Getting the current kernel filename and version"
-  #TODO: Why not just symlink to /boot/vmlinuz
-  # Since our boot partition is FAT, it doesn't support sylminks.
-  #shellcheck disable=SC2012 # We know it's going to be alphanumeric only!
-  mapfile -t KRNL_VERS < <(ls -t /boot/vmlinuz* | sort)
-  log "Found ${#KRNL_VERS[@]} kernel(s)" "${KRNL_VERS[@]}"
-  KRNL=$(echo ${KRNL_VERS[0]##*/} | awk -F "-" '{print $1}')
-  log "Finished Kernel ${KRNL_VERS[0]} installation" "okay" "${KRNL}"
-  mv ${KRNL_VERS[@]} /boot/${KRNL}
-
-  ls -l /boot
+  log "Chanege linux kernel image name to 'vmlinuz'"
+
+  # Rename linux kernel to a fixed name, like we do for any other platform.
+  # We only have one and we should not start multiple versions.
+  # - our OTA update can't currently handle that and it blows up size of /boot and /lib.
+
+  # This rename is safe, because we have only one vmlinuz in /boot
+  mv $(ls /boot/vmlinuz*) /boot/vmlinuz
```

Comment:
Shouldn't this just be `mv /boot/vmlinuz* /boot/vmlinuz`?

Reply:
even simpler when that works, never thought of that